weave/trace/urls.py

```diff
@@ -32,4 +33,21 @@ def leaderboard_path(entity_name: str, project_name: str, object_name: str) -> s
 
 
 def redirect_call(entity_name: str, project_name: str, call_id: str) -> str:
+    # Check if WF_TRACE_SERVER_URL is set (indicating local development)
+    trace_server_url = os.getenv("WF_TRACE_SERVER_URL")
+    if trace_server_url:
+        # Parse the trace server URL to extract the base URL
+        parsed = urlparse(trace_server_url)
+        # Use the host and port from the trace server URL
+        # Default to localhost:9000 if parsing fails or no host is found
+        if parsed.hostname:
+            base_url = f"{parsed.scheme or 'http'}://{parsed.hostname}"
+            if parsed.port:
+                base_url += f":{parsed.port}"
+        else:
+            # Fallback to localhost:9000 if we can't parse the URL properly
+            base_url = "http://localhost:9000"
+        return f"{base_url}/{entity_name}/{quote(project_name)}/{WEAVE_SLUG}/r/call/{call_id}"
+
```

Comment:
I see this
```python
def remote_project_root_url(entity_name: str, project_name: str) -> str:
    return f"{wandb.app_url(env.wandb_base_url())}/{entity_name}/{quote(project_name)}"
```

why isn't this override done inside `env.wandb_base_url()`?

Reply:
I don't think that's part of this change.  Per skimming the code, I think it's been set like this for about a year.

I can take a closer look